lib/irb/easter-egg.rb

```diff
@@ -142,6 +143,7 @@ def render_frame(i)
         end
       end
     ensure
+      print "\e[?25h" # show cursor
```

Comment:
`print "\e[?25l"` is written inside `STDOUT.cooked do end` how about move this to the ensure statement of `STDOUT.cooked do ensure end`?

Reply:
Good point, I've fixed it.